dot/core/service.go

```diff
@@ -69,6 +71,9 @@ type Service struct {
 
 	// Keystore
 	keys *keystore.GlobalKeystore
+
+	runtimeChangedLock sync.RWMutex
+	runtimeChanged     map[byte]chan<- runtime.Version
```

Comment:
rename `runtimeUpdateSubscriptions` or some thing like that?

Reply:
updated to `runtimeUpdateSubscriptions` and `runtimeUpdateSubscriptionsLock`